### PR TITLE
#341: updated service-layer doc

### PR DESCRIPTION
**Pull request by hohwille**
Addresses issue #341 by improving service-layer documentation.

### CODE REVIEW

File: documentation/guide-service-layer.asciidoc [hunk 1]
```diff
@@ -6,32 +6,22 @@ toc::[]
...
-** *Back-end Services* +
-are internal services between Java back-end components typically with different release and deployment cycles (if not Java consider this as an external service).
-** *JS-Client Services* +
-are internal services provided by the Java back-end for JavaScript clients (GUI).
-** *Java-Client Services* +
-are internal services provided by the Java back-end for a native Java client (JavaFx, EclipseRcp, etc.).
+Before you start creating your services you should consider some general design aspects:
 
-The choices for technology and protocols will depend on the type of service. The following table gives a guideline for aspects according to the service types.
+* Do you want to create a https://en.wikipedia.org/wiki/RPC[RPC] service?
```
Comment (maybeec) [inline documentation/guide-service-layer.asciidoc:12]: This link is odd. It does not provide any value as it is not pointing to a concrete content side of wikipedia

**Review by maybeec — CHANGES_REQUESTED** [review]

File: documentation/guide-service-layer.asciidoc (hunk 1 above)
Comment (hohwille) [reply documentation/guide-service-layer.asciidoc:12]: What is odd with wikipedia? This link helps if someone does not know what RPC stands for. What would you like to explain this?

Comment (maybeec) [reply documentation/guide-service-layer.asciidoc:12]: The link you provided does not help to understand what RPC stands for as you simply provide a generic link and wikipedia is asking you to select the content of interest. ;)

Comment (hohwille) [reply documentation/guide-service-layer.asciidoc:12]: Ouch, now I understand. Sorry, that was a C&P mistake. Should have been https://en.wikipedia.org/wiki/Remote_procedure_call
Will fix it asap.

**Review by maybeec — APPROVED** [review]

**Pull request merged**